hkmc2/shared/src/main/scala/hkmc2/semantics/ucs/Normalization.scala

```diff
@@ -294,7 +294,7 @@ class Normalization(using tl: TL)(using Raise, Ctx, State) extends DesugaringBas
     _.flatMap:
       case arg: FlatPattern.Argument.Term => S(arg)
       case FlatPattern.Argument.Pattern(_, pattern) =>
-        error(msg"Pattern argument `${pattern.showDbg}` cannot be used here." -> pattern.toLoc); N
+        error(msg"The pattern argument cannot be used here." -> pattern.toLoc); N
```

Comment:
This is again a badly phrased error.
```suggestion
        error(msg"This `${pattern.describe}` pattern cannot be used as an argument here." -> pattern.toLoc); N
```

(Having a `Pattern#describe` method seems like a good idea for ewrror messages).

Reply:
Also, it seems this error is never exercised (doesn't appear in the diff-tests).

Reply:
It’s exercised here:
https://github.com/hkust-taco/mlscript/blob/7a52f7b998695593f0913554e288bdf992923816/hkmc2/shared/src/test/mlscript/rp/syntax/WrongArguments.mls#L1-L12

Reply:
Ah, right.

Reply:
Fixed in 62ca88236.